include/evmmax/evmmax.hpp

```diff
@@ -0,0 +1,42 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include <intx/intx.hpp>
+
+namespace evmmax
+{
+
+/// The modular arithmetic operations for EVMMAX (EVM Modular Arithmetic Extensions).
+template <typename UintT>
+class ModArith
+{
+public:
+    using uint = UintT;
```

Comment:
Ah, this naming is so confusing.

Reply:
Like what?

Reply:
This alias is not very needed for the implementation.

Reply:
Removed.